### PR TITLE
SOC-1269 when the user is authenticated auth modal is loading regular wiki page

### PR DESCRIPTION
**Pull request by rafalkalinski**
https://wikia-inc.atlassian.net/browse/SOC-1269

### CODE REVIEW

File: server/views/_layouts/modal-empty.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (kvas-damian) [inline server/views/_layouts/modal-empty.hbs]: This directory is shared across all apps, please make that filename suggest it's connected to auth.

File: server/views/auth/desktop/modal-message.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<script>
+	if (window.parent) {
+		window.parent.postMessage({isUserAuthorized: true}, window.parent.location.protocol + '//' + location.hostname);
+	} else {
+		window.location.reload();
```
Comment (kvas-damian) [inline server/views/auth/desktop/modal-message.hbs]: Infinitive loop?

File: server/facets/auth/authView.ts [hunk 3]
```diff
@@ -123,6 +123,25 @@ module authView {
 		}
 		return this.VIEW_TYPE_DESKTOP;
 	}
+
+	export function requestAuthenticated(request: Hapi.Request, reply: any, context: AuthViewContext): Hapi.Response {
```
Comment (bkoval) [inline server/facets/auth/authView.ts]: Could this name be slightly more descriptive? :)

File: server/views/_layouts/modal-empty.hbs [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1, minimal-ui">
+	<link rel="canonical" href="{{canonicalUrl}}" />
+</head>
+<body data-page-type="{{pageType}}">
+{{{content}}}
```
Comment (bkoval) [inline server/views/_layouts/modal-empty.hbs]: I would consider nesting it by one tab, what do you think?

File: server/views/auth/desktop/modal-message.hbs (hunk 2 above)
Comment (bkoval) [inline server/views/auth/desktop/modal-message.hbs]: Is this even needed? We only need postMessage for modal implementations.

**Comment by kvas-damian** [comment]
LGTM :japanese_ogre:

**Comment by bkoval** [comment]
LGTM

**Comment by rafalkalinski** [comment]
https://github.com/Wikia/mercury/pull/1355

**Pull request closed without merging**